src/elements/content-sidebar/activity-feed/comment-form/CommentForm.js

```diff
@@ -129,7 +134,7 @@ class CommentForm extends React.Component<Props, State> {
             <Media className={inputContainerClassNames}>
                 {!isEditing && (
                     <Media.Figure className="bcs-CommentForm-avatar">
-                        <Avatar getAvatarUrl={getAvatarUrl} user={user} />
+                        {renderAvatar(<Avatar getAvatarUrl={getAvatarUrl} user={user} />)}
```

Comment:
Do you have a screenshot of what this looks like?

Reply:
I have and it's in the PR for Comments tooltip, which actually pass this property to `PreviewContent`. I'll gladly share it with you, but I was told to not post links to our internal Box systems in this public repo.

And then I don't feel that's connected to this PR. This PR adds a property on general purpose component `PreviewContent`, which can be possibly reused in other contexts than just the tooltip I'm now implementing. So it should be examined in this general context as well and not in the context of the tooltip I'm working on. At least that's how I see it. Please tell me if you feel otherwise.